### PR TITLE
[storage] short state key prefix

### PR DESCRIPTION
**Pull request by msmouse**
## Motivation

One byte should be good enough for this purpose.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md#pull-requests)?

Y

## Test Plan

existing coverage

### CODE REVIEW

File: types/src/state_store/state_key.rs [hunk 1]
```diff
@@ -26,36 +24,42 @@ pub enum StateKey {
...
+                (StateKeyPrefix::AccountAddress, account_address.to_vec())
             }
             StateKey::AccessPath(access_path) => {
-                let mut account_access_path_prefix =
-                    ACCOUNT_ACCESS_PATH_KEY_PREFIX.as_bytes().to_vec();
-                account_access_path_prefix.extend(access_path.address.to_vec());
-                RawStateKey {
-                    bytes: account_access_path_prefix,
-                }
+                // TODO: include the path part
```
Comment (msmouse) [inline types/src/state_store/state_key.rs:54]: @sitalkedia  was it deliberate that this doesn't include the `path` part?

Comment (sitalkedia) [reply types/src/state_store/state_key.rs:54]: @msmouse - This was an oversight. As of now, this path is not exercised so this is not an issue, but let's include the prefix in this diff if you are okay.

Comment (sitalkedia) [reply types/src/state_store/state_key.rs:54]: Actually, it seems like it was included earlier. Any reason you choose to remove it?

Comment (msmouse) [reply types/src/state_store/state_key.rs:54]: it had ap.address, but not ap.path?

Comment (sitalkedia) [reply types/src/state_store/state_key.rs]: Ah, right. This is an oversight.

**Review by sitalkedia — APPROVED** [review]

**Pull request merged**